include/klee/ArrayExprOptimizer.h

```diff
@@ -0,0 +1,100 @@
+//===-- ArrayExprOptimizer.h ----------------------------------------------===//
+//
+//                     The KLEE Symbolic Virtual Machine
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef KLEE_EXPROPTIMIZER_H
+#define KLEE_EXPROPTIMIZER_H
+
+#include <cassert>
+#include <iterator>
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "Expr.h"
+#include "ExprBuilder.h"
+#include "Constraints.h"
+#include "Internal/Support/ErrorHandling.h"
+#include "util/ArrayExprVisitor.h"
+#include "util/Assignment.h"
+#include "util/Ref.h"
+#include "klee/AssignmentGenerator.h"
+#include "klee/ExecutionState.h"
+
+#include "llvm/Support/TimeValue.h"
+#include "klee/Internal/System/Time.h"
```

Comment:
And also these two includes. I can't see why they are needed here and the LLVM 4.0 build fails because of these.